src/app/page.tsx

```diff
@@ -1,84 +1,108 @@
-'use client'
-import { SocialPlatform } from '@/types';
-import download from 'downloadjs';
-import { toPng } from 'html-to-image';
-import Image from 'next/image';
+"use client";
```

Comment:
btw I think my VSCode just did a massive refactor on the page... not sure its what you want though 🤔  let me know I can remove pretty much all of them and just leave the code, or if you want just part of the refactor, I can help to clean some of them.